src/languageclient/reloadSchemas.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '../extensionVariables';
+import { ReloadShemasRequest } from './ReloadSchemasRequest';
+
+export async function reloadSchemas(): Promise<void> {
+
+    ext.outputChannel.appendLine("Reloading schemas...");
+
+    if (ext.languageServerClient) {
```

Comment:
How should we handle when the language server hasn't started yet?  It should be the same as when the lang server has started but isn't ready yet (you'll get an error sending the notification before the await ext.languageServerClient.onReady() line has run).  I'd say two possibilities:
1) throw an error, tell the user to wait until lang server is ready
2) set a flag to send the notification as soon as it's ready

I'm okay with either.  Ideally, I think we should wait to assign ext.languageServerClient until after the await onReady call completes, though, so we avoid this ambiguity - we either have a lang server and it's ready, or we don't have it yet.

Reply:
I'm also hesitating on whether to expose ext.languageServerClient.  I'm thinking maybe instead exposing a sendLanguageServerNotification<T>(type, args, waitForStart: boolean) function from the startArmLanguageServer.ts module.  Thoughts?

Reply:
I was following Razor model (more or less). The do wrap the LanguageClient field into RazorLanguageServerClient, but they do expose an instance of RazorLanguageServerClient to the rest of the extension. I'd be OK either way.

Reply:
I'm not too worried about it either way for this change, although it if were me I'd probably not expose the lang server client itself since it's really an implementation detail.  Either I'd expose an interface with specific fields or more likely just expose functions from the module.

Reply:
I left it as is for now since we are a bit pressed for time, but we can chat later and clean up as we see fit, possibly creating our own ArmLanguageServerClient class encapsulating LanguageClient kind of like Razor does.

Reply:
Then I'd suggest you throw an error if langserver==null rather than just doing nothing.